src/Task/SecurityCheckerEnlightn.php

```diff
@@ -50,6 +52,11 @@ public function run(ContextInterface $context): TaskResultInterface
         $arguments = $this->processBuilder->createArgumentsForCommand('security-checker');
         $arguments->add('security:check');
         $arguments->addOptionalArgument('%s', $config['lockfile']);
+        if (!empty($config['allow_list'])) {
```

Comment:
You should use `$arguments->addArgumentArrayWithSeparatedValue()` for this specific purpose (instead of the empty + foreach + broken optional argument statement)